src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -871,3 +953,23 @@ julia> nvertices(cp)
 ```
 """
 cyclic_polytope(d::Int, n::Int) = Polyhedron(Polymake.polytope.cyclic(d, n))
+
+# random constructions
+
+@doc Markdown.doc"""
+    rand_spherical_polytope(d::Int, n::Int)
+
+Construct the convex hull of $n$ points on the unit sphere in $\mathbb{R}^d$.
```

Comment:
$n$ rational points approximately on the unit sphere...?

Reply:
That one is more subtle!  polymake has the full drill; Oscar needs to learn it.  I'll write a bit about this in our chapter.  We keep the code (and doc) for now; to be improved later.